src/__tests__/test-files/documents/sap.ref-app-example.json

```diff
@@ -0,0 +1,388 @@
+{
+  "openResourceDiscovery": "1.6",
+  "policyLevel": "sap:core:v1",
+  "description": "This is an example ORD document which is unprotected and openly accessible.",
+  "describedSystemInstance": {
+    "baseUrl": "https://ord.example.com/ord/v1"
+  },
+  "products": [
+    {
+      "ordId": "sap.xref:product:ord-reference-app:",
+      "title": "ORD Reference App",
+      "vendor": "sap:vendor:SAP:",
+      "shortDescription": "Open Resource Discovery (ORD) Reference Application"
+    }
+  ],
+  "packages": [
+    {
+      "ordId": "sap.xref:package:ord-reference-app-apis:v1",
+      "title": "Reference Application",
+      "shortDescription": "This reference package demonstrates how API resources can be published on Business Accelerator Hub",
+      "description": "This reference application demonstrates how different resources can be published on Business Accelerator Hub",
+      "version": "1.0.0",
+      "policyLevel": "sap:core:v1",
+      "partOfProducts": ["sap.xref:product:ord-reference-app:"],
+      "vendor": "sap:vendor:SAP:",
+      "tags": ["reference application"],
+      "packageLinks": [
+        {
+          "type": "license",
+          "url": "https://ord.example.com/open-resource-discovery-reference-application/blob/master/LICENSE.txt"
```

Comment:
Please replace `master` with `main` in this link everywhere to make "Inclusive Language" check happy.